public/app/features/visualization/data-hover/ExemplarHoverView.tsx

```diff
@@ -79,6 +80,8 @@ const getStyles = (theme: GrafanaTheme2, padding = 0) => {
       gap: 4,
       borderTop: `1px solid ${theme.colors.border.medium}`,
       padding: theme.spacing(1),
+      overflowY: 'scroll',
```

Comment:
this should probably be `'auto'` so you dont get a forced scrollbar when the contents are < maxHeight

Reply:
Thanks, modified.